pkg/skuba/actions/validate/validate.go

```diff
@@ -19,16 +19,25 @@ package validate
 
 import (
 	"fmt"
+	"regexp"
 
 	"github.com/SUSE/skuba/pkg/skuba"
 )
 
+var nodeNameRegex = regexp.MustCompile(`^[a-zA-Z0-9]([-a-zA-Z0-9]*[a-zA-Z0-9])?(\.[a-zA-Z0-9]([-a-zA-Z0-9]*[a-zA-Z0-9])?)*$`)
```

Comment:
Does uppercase char work without issues? do you verify by e2e?

Reply:
Yes, I used `CaaSP-master-mkravec-0` and `CaaSP-worker-mkravec-0` and cluster built fine.
I think skuba lowercases them internally. I am not sure if we should allow `A-Z` if we rewrite them anyway.

```
skuba node bootstrap --user sles --sudo --target $addr CaaSP-master-mkravec-0
skuba node join -s -u sles -t $addr -r worker CaaSP-worker-mkravec-0

skuba cluster status
** This is an UNTAGGED version and NOT intended for production usage. **
NAME                     OS-IMAGE                              KERNEL-VERSION           KUBELET-VERSION   CONTAINER-RUNTIME   HAS-UPDATES   HAS-DISRUPTIVE-UPDATES
caasp-master-mkravec-0   SUSE Linux Enterprise Server 15 SP1   4.12.14-197.18-default   v1.15.2           cri-o://1.15.0      <none>        <none>
caasp-worker-mkravec-0   SUSE Linux Enterprise Server 15 SP1   4.12.14-197.18-default   v1.15.2           cri-o://1.15.0      <none>        <none>
```

Reply:
Relevant information: https://github.com/kubernetes/kubernetes/issues/71140#issuecomment-441703265 and https://github.com/kubernetes/kubernetes/blob/c8c9d2e21ce9afe52ac8ace64f96a469891dc724/pkg/util/node/node.go#L49-L66

I think it's safer if we don't allow for upper case names.

Reply:
Agreed, so suggest to just follow the same pattern used in upstream without uppercase.

Reply:
done